src/Corax/IndexEntryBuilder.cs

```diff
@@ -30,17 +30,26 @@ public enum IndexEntryFieldType : byte
     {
         Null = 0, // Helper for list writer.
         None = 1,
-        
+
         Tuple = 1 << 2,
         List = 1 << 3,
         Raw = 1 << 4,
+        Special = 1 << 5,
```

Comment:
Special, or spatial :-) ?

Reply:
We will be out of flags here soon. I will change IndexEntryFieldType from `byte` into `uint`.